.github/workflows/gradle.yml

```diff
@@ -4,8 +4,9 @@
 name: Java CI with Gradle
 
 # Triggers the workflow on push or pull request events (on every branch)
-on: [push, pull_request]
-
+on:
```

Comment:
why did you remove the pull_request hook? Also, the v1.2.1 branch can be removed.

Reply:
they didn't work